erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -43,8 +43,8 @@ const BaseStack = (specInput) => {
   specBase.remoteDescriptionSet = false;
 
   that.mediaConstraints = {
-    offerToReceiveVideo: (that.video !== undefined && that.video !== false),
-    offerToReceiveAudio: (that.audio !== undefined && that.audio !== false),
+    offerToReceiveVideo: true,
```

Comment:
This was problematic at some point with firefox. We should try it and move it to Chrome's stack if that's the case.

Reply:
good catch!